receiver/githubreceiver/README.md

```diff
@@ -0,0 +1,56 @@
+# GitHub Receiver
+
+Receives logs from [GitHub](https://github.com/)
+via the [GitHub API](https://docs.github.com/en/rest?apiVersion=2022-11-28).
+
+## Minimum Agent Versions
+
+- Introduced: [v1.59.0](https://github.com/observIQ/bindplane-agent/releases/tag/v1.59.0)
+
+## Supported Pipelines
+
+- Metrics
+- Logs
+
+## How It Works
+
+1. The user configures their instance of GitHub Enterprise to enable monitoring of audit logs.
+2. The user configures this receiver in a pipeline.
+3. The user configures a supported component to route telemetry from this receiver.
+
+## Prerequisites
+
+- Created instance of GitHub with the following subscriptions: GitHub Enterprise Cloud
+- Access to an admin account for any enterprise, organization, or repo required for audit logs.
+
+## Configuration
+
+| Field         | Type          | Default | Requried | Description                                                                                                                                                                                                                                                                                       |
+| ------------- | ------------- | ------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| AccessToken   | string        |         | `true`   | Access token is required for audit log generation. Grants access to enterprise and organization if access token shows admin role. https://docs.github.com/en/enterprise-cloud@latest/apps/creating-github-apps/(authenticating-with-a-github-app/generating-a-user-access-token-for-a-github-app) |
```

Comment:
Use a configopaque.String for sensitive fields instead of just a normal string. 
https://github.com/open-telemetry/opentelemetry-collector/blob/e375da45118c3b82edb6adf556c8ecbc765e7aa1/config/configopaque/opaque.go#L12
Shoutout Brandon for teaching me about this